iCubGenova04/icub_right_leg.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+    <robot name="iCubGenova04" build="1" portprefix="icub" xmlns:xi="http://www.w3.org/2001/XInclude">
+        <params>
+    <xi:include href="hardware/electronics/pc104.xml" />
+        </params>
+        
+    <devices>
+    <!-- HEAD -->
+    <!--<xi:include href="./hardware/motorControl/head-eb20-j0_1-mc.xml" />  
+    <xi:include href="./hardware/motorControl/head-eb21-j2_5-mc.xml" /> 
+    <xi:include href="./wrappers/motorControl/head-mc_wrapper.xml" /> -->
+ 
+    <!-- TORSO --> 
+    <xi:include href="hardware/motorControl/torso-eb5-j0_2-mc.xml" />
+    <xi:include href="wrappers/motorControl/torso-mc_wrapper.xml" />
```

Comment:
This file is called ``right_leg`` but it starts the whole robot. Maybe a different name?

Reply:
Actually that file is not required anymore. The robot can start with any of the other scripts. So I'll just remove it. I'll wait for the other comments to push the update.

Reply:
sorry @S-Dafarra the PR has been merged without the change on the cleanup removal of the `icub_right_leg.xml` script. I will do it in my next PR this week.